storage/remote/store_test.go

```diff
@@ -156,6 +156,10 @@ func (a testClient) ListSealsForHeight(ctx context.Context, in *archive.ListSeal
 }
 
 func Test_SimulatedMainnetTransaction(t *testing.T) {
+
+	// TODO: enable
+	t.SkipNow()
```

Comment:
This test and below, rely on a mainnet contract, that is not compatible with stable cadence.